tests/snapshots/readback_lnet__match.hvm.snap

```diff
@@ -2,4 +2,4 @@
 source: tests/golden_tests.rs
 input_file: tests/golden_tests/readback_lnet/match.hvm
 ---
-let b = 1; match b { 0: λa a; +: b-1 }
+let b = 1; match b { 0: λa a; +b-1: b-1 }
```

Comment:
Shouldn't this `b-1` be omitted as it is not a valid syntax?

Reply:
Except for the invalid name, it is valid syntax. Do you think we should just leave it with a generic variable name?

Reply:
What do you mean it is valid syntax? The `+` arm of a native match can only be `+: body`, if you put any name there the parser will give an error

Reply:
What i mean is that: Shouldn't the readback contain only valid syntax that just by copying it and pasting into a file it should be parseable without problems?

Reply:
It should accept both versions, like we do for adt constructors. If right now it doesn't then we should fix it